integration_tests/test_suites/k8s-test-suite/tests/test_external_asset.py

```diff
@@ -169,12 +169,12 @@ def test_use_excute_k8s_job(namespace, cluster_provider):
     def number_y_job(context: AssetExecutionContext):
         core_api = kubernetes.client.CoreV1Api()
         reader = K8sPodLogsMessageReader()
-        with ext_protocol(
+        with pipes_protocol(
```

Comment:
worth considering names for this fn that make sense since its opening a context manager:
`open_pipes`
`open_pipes_session`
`start_pipes_session`

Reply:
I like `open_pipes_session` to mirror `open` in Python. 👍🏻